Dockerfile

```diff
@@ -1,47 +1,39 @@
-FROM postgres:11
-
-ARG DEBIAN_FRONTEND=noninteractive
-ARG BUILD_DEPS='gcc git libffi-dev libssl-dev python3-dev python3-pip python3-wheel'
-
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-        $BUILD_DEPS \
-        gosu \
-        lzop \
-        libpq-dev \
-        pv \
-        python3 \
-        util-linux \
-        # swift package needs pkg_resources and setuptools
-        python3-pkg-resources \
-        python3-setuptools \
-        python3-pip && \
-    ln -sf /usr/bin/python3 /usr/bin/python && \
-    ln -sf /usr/bin/pip3 /usr/bin/pip
-
-# setuptools from ubuntu archives is too old for googleapis-common-protos
-RUN pip install --upgrade setuptools && \
-    pip install --disable-pip-version-check --no-cache-dir \
-        envdir==1.0.1 \
-        wal-e[aws,azure,google,swift]==1.1.0 \
-        gcloud==0.18.3 \
-        oauth2client==4.1.3 \
-        azure-storage==0.20.0
-
-# cleanup
-RUN apt-get purge -y --auto-remove $BUILD_DEPS && \
-    apt-get autoremove -y && \
-    apt-get clean -y
+FROM postgres:11-alpine
```

Comment:
Should we use a more specific tag? Like `11.0-alpine` or `11.1-alpine`? (No idea if Postgres knows semver.)

Reply:
The version after postgres10 changed the naming policy, so postgres11.x is backward compatible.

Reply:
Sounds like postgres:11-alpine will be alright then.

Reply:
Should we use a 11.x version that is higher than 11.0 then or does 11 have the fixes in the later versions of 11.x?

Reply:
The exact point postgres:11-alpine can be, it is just a link, pointing to the latest postgres:11.x-alpine. In this way, we will use the latest image every time.

Reply:
Good :+1: